setup.py

```diff
@@ -19,13 +19,23 @@
 
 import io
 import os
+import re
 import setuptools
 
-# The following line is parsed by CI scripts (see .circleci/config.yml)
-# and by release.py. Please keep the format.
-current_version = '0.15.5'
-
 cur_dir = os.path.realpath(os.path.dirname(__file__))
+
+# version is defined in Dockerfile: ENV ARTMAN_VERSION version
+current_version = None
+with io.open('%s/Dockerfile' % cur_dir) as dockerfile:
+    for line in dockerfile:
+        match = re.match('^ENV ARTMAN_VERSION (\S+)$', line)
+        if match:
+            current_version = match.group(1)
```

Comment:
Do we want to `break`?

Reply:
We can (in hope that no one will define version twice :) )